core-bundle/src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -79,11 +79,17 @@ public function output($blnCheckRequest=false)
 	 *
 	 * @return Response The response object
 	 */
-	public function getResponse($blnCheckRequest=false)
+	public function getResponse($blnCheckRequest=false, $blnForceCacheHeaders=false)
```

Comment:
What's this new parameter for?

Reply:
I guess @aschempp forgot to add it to the if condition (see [my suggestion](https://github.com/contao/contao/pull/718/files#r353684908)).

Reply:
fixed in 2dbedf0